src/HealthChecks.Elasticsearch/ElasticsearchHealthCheck.cs

```diff
@@ -26,6 +26,11 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                 {
                     var settings = new ConnectionSettings(new Uri(_options.Uri));
 
+                    if (_options.RequestTimeout.HasValue)
+                    {
+                        settings.RequestTimeout(_options.RequestTimeout.Value);

```

Comment:
Why not `settings = settings.RequestTimeout(_options.RequestTimeout.Value);` as all cases below ?